sig-windows/CONTRIBUTING.md

```diff
@@ -95,6 +95,16 @@ If filing a bug, please include detailed information about how to reproduce the
 
 Logs are an important element of troubleshooting issues in Kubernetes. Make sure to include them any time you seek troubleshooting assistance from other contributors.
 
+### Collecting kubelet and kube-proxy Logs
+There are a few different ways to run the node binaries (kubelet, kube-proxy, etc) and the execution method also dictates how logs will be collected. [Issue 75319](https://github.com/kubernetes/kubernetes/issues/75319) tracks pending work for better log management on Windows (for example using the Windows Event Log for higher log throughput and log rotation). If you end up logging to a file, you can use [fluentd](https://docs.fluentd.org/v0.12/articles/windows) or Splunk to ship logs to a syslog server for search and analytics.
```

Comment:
For Splunk, may be a link to the Windows data collector may be useful: https://docs.splunk.com/Documentation/Splunk/7.2.4/Data/HowtogetWindowsdataintoSplunk